### PR TITLE
dm-2504-a patch to categories

### PR DESCRIPTION
**Pull request by bradjohnson92008**
Remove not ! in front of present? statements in admin panel.

### JIRA issue link
https://agile6.atlassian.net/browse/DM-2504

## Description - what does this code do?
Updaes boolean logic for parent category display in admin panel.

## Testing done - how did you test it/steps on how can another person can test it 
Log in as Admin
go to the Admin Panel /admin
click on Categories
**You should see categories "Parent Category" column populated for almost all categories.
image

## Definition of done
- [ ] Unit tests written (if applicable)
- [ ] e2e/accessibility tests written (if applicable)
- [ ] Events are logged appropriately
- [ ] Documentation has been updated, if applicable
- [ ] A link has been provided to the originating JIRA issue
- [ ] No sensitive information (i.e. PII/credentials/internal URLs/etc.) is captured in logging, hardcoded, or specs

### CODE REVIEW

File: app/services/save_practice_service.rb [hunk 1]
```diff
@@ -190,16 +189,17 @@ def update_category_practices
...
           parent_cat_id_param = category[:parent_category_id]
           # if the 'other' category has not yet been created, the :parent_category_id will be a string, so we need to find the corresponding parent category
-          parent_cat_id = parent_cat_id_param.to_i === 0 ? Category.where('lower(name) = ?', parent_cat_id_param.downcase).first.id : parent_cat_id_param
+          parent_cat_id = parent_cat_id_param.to_i === 0 ? Category.get_category_by_name(parent_cat_id_param).first.id : parent_cat_id_param
           unless name == ""
-            if destroy == 'false' && id.blank?
-              cate = Category.find_by(name: name.strip.downcase, is_other: true, parent_category_id: parent_cat_id)
-              cate = Category.create(name: name.strip.downcase, is_other: true, parent_category_id: parent_cat_id) unless cate.present?
+            if destroy != '1' && id.blank?
+              cate = Category.find_by(name: name.strip, is_other: true, parent_category_id: parent_cat_id)
```
Comment (emyl3) [inline app/services/save_practice_service.rb:195]: I think we should do something like:
``` 
cat_name = name.strip.downcase
cate = Category.where('lower(name) = ?', cat_name).where(is_other: true).first
```

Comment (bradjohnson92008) [reply app/services/save_practice_service.rb:195]: Cool - so ... this is the change right..
image

Comment (emyl3) [reply app/services/save_practice_service.rb:195]: haha please ignore https://agilesix.slack.com/archives/C010LQ2V3MZ/p1626207673054800?thread_ts=1626201526.048100&cid=C010LQ2V3MZ 😅

**Review by emyl3 — APPROVED** [review]
Looks good to me!

**Review by JoshingYou1 — APPROVED** [review]
Works as intended! Great job @bradjohnson92008 and @emyl3!

**Pull request merged**